apps/web/src/views/AddLiquidityV3/index.tsx

```diff
@@ -403,12 +405,32 @@ export function AddLiquidityV3Layout({
 
   const title = SELECTOR_TYPE_T[selectType] || t('Add Liquidity')
 
+  const lpTokens = useStableSwapPairs()
+
+  const backToLink = useMemo(() => {
+    const isIncrease = Boolean(router.query.increase)
```

Comment:
Can check from the route path instead of adding a query parameter? Plus shall we use router.back if possible?

Reply:
Can we check if there's history? If there is then we use router.back. Otherwise we just assemble the url and navigate